ansible/hedgehog_deployment_script.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+#todo use docker filter with regex
+#MGMT_CONTAINER=`docker ps -f ancestor=docker-sonic-mgmt-hedgehog:master --format {{.Names}}`
+
+MGMT_CONTAINER=`docker ps | awk '/docker-sonic-mgmt/ { print $NF }'`
+CURRENT_TOPO_CONTAINER=`docker ps | awk '/sonicdev-microsoft.azurecr.io/ { print $NF }'`
+SONIC_MGMT_REPO="sonic-mgmt_dev_202205"
+
+pre_check () {
+  echo "Pre check"
+  if [ -z "$MGMT_CONTAINER" ]; then
+    echo "No sonic-mgmt container is running. Exit 1."
+    exit 1
+  else
+    echo "The sonit-mgmt container is running: $MGMT_CONTAINER"
+  fi
+}
+
+remove_topo () {
```

Comment:
There is `veos-vm/cleanup.sh` script. Should we use it here?

Reply:
didn't know about that script, 
I just followed the instruction
https://github.com/githedgehog/sonic-mgmt/blob/master/docs/testbed/README.testbed.VsSetup.md#restoreremove-the-testing-environment

Reply:
I just checked this script, it is more than we need.
I mean this script stop all VMs, remove VM disks, stop/remove all containers/images

I propose do not use this script. just to save time for deploying

Reply:
extend this script with `cleanup.sh` in separate pr